card/lib/cardio/commands.rb

```diff
@@ -0,0 +1,119 @@
+# add method in? to Object class
+require "active_support/core_ext/object/inclusion"
+require "cardio"
+
+require "rake"
+def load_rake_tasks
+  require "./config/environment"
+  require "rake"
+  Cardio::Application.load_tasks
+end
+
+module Cardio
+  module Commands
```

Comment:
These changes leave a TON of duplicate code in decko and card.  Anything that's already in cardio should be includable in decko.